### DISCUSSION TITLE
如何开发环境时打一个测试环境和正式生成环境的包

### DISCUSSION THREAD

**Discussion by iTulip331** [alex8088/electron-vite#416] [Q&A]
例如electron-builder yml中publish的地址我如何通过环境变量来区别

**Comment by alex8088** [comment]
可以多个配置文件，打包时指定不同的配置文件，详细看electron-builder的文档

**Reply by iTulip331** [reply]
主要是由于我生产环境时，分测试服务器和正式服务器相关的地址不同，是不是加一个测试环境变量呢，scripts里的命令如何修改呢

**Reply by takegine** [reply]
> 主要是由于我生产环境时，分测试服务器和正式服务器相关的地址不同，是不是加一个测试环境变量呢，scripts里的命令如何修改呢

is.dev 可以区分你是 electron-vite dev 还是 electron-vite preview(build)，
